File: Java/URI1008.java
import java.util.Scanner;

public class URI1008 {
	public static void main(String[] args) {
		Scanner entrada = new Scanner(System.in);
		int num = entrada.nextInt();
		int horas = entrada.nextInt();
		double recHor = entrada.nextDouble();
		double salario = horas * recHor;
		System.out.printf("NUMBER = %d\nSALARY = U$ %.2f\n", num, salario);
		
	}
}

File: C/uri1008.c
#include <stdio.h>

void main(){
    int num, qhoras;
    double valor;

    scanf("%d %d %lf", &num, &qhoras, &valor);
    printf("NUMBER = %d\nSALARY = U$ %.2lf\n", num, (qhoras*valor));
}